### ISSUE TITLE
Status of Repository?

### ISSUE THREAD

**Issue by rethab** [github-rs/github-rs#177]
hi, what is the status of this repository? I see the build is failing and several PRs are unmerged w/o any comments.

If there's anything specific I could help with to breathe some life into this, please let me know.

**Comment by rostyslavkhaniukov** [comment]
@Dylan-DPC, what about this question?

**Comment by rostyslavkhaniukov** [comment]
or @brndnmtthws

**Comment by SirWindfield** [comment]
I'd be willing to be an active maintainer in the sense that I'd take a look at PRs and issues and merge accordingly. Given the number of outstanding PRs there are a lot of people relying on the crate :)

**Comment by jakechv** [comment]
Is there an active fork for this crate at all? I'd be happy to contribute to maintenance as well.

**Comment by SirWindfield** [comment]
I don't think so. It seems that the project isn't being used a lot anymore. It doesn't get pulled in that often. There are other projects that fill in the gap, such as [hubcaps](https://lib.rs/crates/hubcaps), [octocrab](https://github.com/xampprocky/octocrab).
